test/integration/driver.go

```diff
@@ -90,7 +91,7 @@ func init() {
 		"Docker hub")
 	flag.StringVarP(&tag, "tag", "t", "",
 		"Docker tag")
-	flag.StringVarP(&mixerTag, "mixerTag", "", "latest",
+	flag.StringVarP(&mixerTag, "mixerTag", "", "",
```

Comment:
can you put SHA here too? I know it's duplication but I use both scripts e2e and driver.go for testing. driver.go bypasses docker image creation. We didn't find a good way to get rid of the shell script.